books/kestrel/number-theory/tonelli-shanks.lisp

```diff
@@ -185,25 +188,24 @@
 ;; that can't happen)
 
 ;; Future work: prove correctness; improve guards
```

Comment:
If the guards are good enough now, we can take out the "; improve guards" future work